File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())

def isPrime(n):
    if (n == 1):
        return False
    count = 0
    for i in range(2,n):
        if (n % i == 0):
            return False
    return True


input_list = list(map(int,input().split()))

count = 0
for number in input_list:
    if (isPrime(number)):
        count += 1

print(count)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.js
const fs = require('fs');
let input = fs
    .readFileSync('/dev/stdin')
    .toString()
    .trim()
    .split('\n')
    .map((item) => item.split(' ').map(Number));

const N = input[0][0];
const n_list = input[1];

const calcaulateMinority = (number) => {
    if (number === 1) return false;
    for (let i = 2; i <= Math.sqrt(number); i++) {
        if (number % i === 0) {
            return false;
        }
    }
    return true;
};
let result = 0;
for (const num of n_list) {
    if (calcaulateMinority(num)) result += 1;
}
console.log(result);
